Makefile

```diff
@@ -241,9 +241,13 @@ pull:
 	for image in $(_DOCKER_PULLED_IMAGES); do docker pull $$image; done
 	touch $@
 
+ifdef OUTSIDE_EPFL
+_OUTSIDE_EPFL_DOCKER_BUILD_ARGS:=--build-arg INSTALL_AUTO_FLAGS=--exclude=wp-media-folder
```

Comment:
Cette partie est dupliquée avec travis.yml dans wp-ops. On pourrait imaginer que dans une prochaine PR, je déplace dans install-plugins-and-themes.py la connaissance du détail que wp-media-folder est inaccessible hors de l'EPFL.